src/libs/ajax/workflows-app/CbasMockResponses.js

```diff
@@ -0,0 +1,591 @@
+import _ from 'lodash/fp';
```

Comment:
Worth trying to condense this at all into reusable variables? I'm also not sure how much of this is similar to the mock data already present in `src/workflows-app`